atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -132,11 +136,17 @@ public int execute(final AtlasDbServices services) {
 
             while (startRow.isPresent()) {
                 Stopwatch watch = Stopwatch.createStarted();
-                SweepResults results = sweepRunner.run(table, batchSize, cellBatchSize, startRow.get());
-                printer.info("Swept from {} to {} in table {} in {} ms, examined {} unique cells, deleted {} stale versions of those cells.",
+
+                SweepResults results = dryRun
+                        ? sweepRunner.dryRun(table, batchSize, cellBatchSize, startRow.get())
+                        : sweepRunner.run(table, batchSize, cellBatchSize, startRow.get());
+                printer.info(
+                        "Swept from {} to {} in table {} in {} ms, examined {} unique cells, {}deleted {} stale versions of those cells.",
                         encodeStartRow(startRow), encodeEndRow(results.getNextStartRow()),
                         table, watch.elapsed(TimeUnit.MILLISECONDS),
-                        results.getCellsExamined(), results.getCellsDeleted());
+                        results.getCellsExamined(),
+                        dryRun ? "would have " : "",
+                        results.getCellsDeleted());
                 startRow = results.getNextStartRow();
```

Comment:
Sadly it's not tractable to instrument that kind of testing, with the voodoo currently contained in TestSweepCommand.

One flag though is that updating the priority table should *not* be the job of SweepCommand.execute (in case, the *only* thing that method should be doing is validating args and executing the sweep command). For now, I'm wrapping that part in an `if` statement.